src/ol_infrastructure/applications/mitlearn/__main__.py

```diff
@@ -371,9 +380,13 @@
     type="kv-v2",
     options={"version": 2},
     description="Storage of configuration secrets used by MIT-Open",
-    opts=ResourceOptions(delete_before_replace=True),
+    opts=ResourceOptions(
+        delete_before_replace=True, depends_on=[mitopen_vault_iam_role]
+    ),
 )
 
+# There is a reason, I think, why these are still at `bridge/secrets/mitopen`
+# and not `bridge/secrets/mitlearn` -- Open Discussions
```

Comment:
Not a good reason. We should rename it. There are many other lingering renaming tasks that we should try to address. Anything that doesn't require downtime should be included in this work.

Reply:
I accomplished this via a symbolic link because I don't want to break anything I don't have to break and I also don't want to spend time digging through the odl-open-discussions stack and refactoring in there.